### PR TITLE
ReadMe Review

### PR DESCRIPTION
**Pull request by helenktsai**
- [x] Apply format template with link to component documentation
- [x] Ensure no dead links

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,32 @@
 # NServiceBus.RavenDB
 
-The official [NServiceBus](https://github.com/Particular/NServiceBus) persistence implementation for [RavenDB.](https://ravendb.net/)
+NServiceBus.RavenDB is the official NServiceBus persistence implementation for [RavenDB](https://ravendb.net/).
```
Comment (soujay) [inline README.md:3]:
```suggestion
NServiceBus.RavenDB is the official [NServiceBus](https://github.com/Particular/NServiceBus) persistence implementation for [RavenDB](https://ravendb.net/).
```

To remain consistent with others like https://github.com/Particular/NServiceBus.Gateway.RavenDB/pull/517/files, have the NServicebus link to the repo

File: README.md [hunk 2]
```diff
@@ -1,27 +1,32 @@
...
+The tests can be run with RavenDB servers hosted on a Docker container.
 
 ## CI Workflow
 
 The [CI workflow](/.github/workflows/ci.yml) requires two secrets unique to RavenDB to be defined both as Actions and Dependabot secrets:
 
-* RAVENDB_LICENSE: A RavenDB development license, expressed as JSON, but all on one line, with escaped quotes `\"`
-* PASSPHRASE: Any phrase used to encrypt the connection information between steps in the CI
+1. RAVENDB_LICENSE: A RavenDB development license, expressed as JSON, but all on one line, with escaped quotes `\"`
+1. PASSPHRASE: Any phrase used to encrypt the connection information between steps in the CI
```
Comment (soujay) [inline README.md:23]: Any phrase used to encrypt the connection information between steps in the CI
- How do we do this in our current ci.yml?

File: README.md (hunk 1 above)
Comment (helenktsai) [reply README.md:3]: Alas, I wasn't consistent with this 😬 But I don't mind having it for sure.a

File: README.md (hunk 2 above)
Comment (helenktsai) [reply README.md:23]: Good catch, looks like it's obsolete, I'll remove it.

**Review by soujay — APPROVED** [review]

**Pull request merged**